src/models/u256_decimal.rs

```diff
@@ -0,0 +1,92 @@
+use primitive_types::U256;
+use serde::{de, Deserializer, Serializer};
+use serde_with::{DeserializeAs, SerializeAs};
+use std::fmt;
+
+// Code copied from here: https://github.com/cowprotocol/services/blob/main/crates/model/src/u256_decimal.rs
+// It was copied, as we prefer to not depend on such a big project.
```

Comment:
👍 I think its super reasonable to not include a separate crate for this small module and just copy it in here. After all, its just glue code and not logic.